minio/mcs
# Add admin heal api and ui

<img width="1549" alt="Screen Shot 2020-05-22 at 5 59 36 PM" src="https://user-images.githubusercontent.com/11819101/82718221-0c8e6300-9c56-11ea-9f12-f0e54d8ccedf.png">

Test Steps:
- On UI a new item on the Menu should appear for `Heal`
- A server should be running in distributed mode you can do `minio server ./dat{1...4}`
- Upload some files to a bucket
- Delete one "disk" `rm -rf ./dat4` 
- Start healing, with recursive either by selecting the bucket (optional)

> Tests missing.
